wom/src/main/scala/wom/graph/InstantiatedExpression.scala

```diff
@@ -1,15 +1,11 @@
 package wom.graph
 
-import cats.instances.list._
-import cats.syntax.traverse._
-import cats.syntax.validated._
-import cats.data.Validated.Valid
 import lenthall.validation.ErrorOr.ErrorOr
 import wdl.types.WdlType
 import wom.expression.WomExpression
-import wom.graph.GraphNodePort.{ConnectedInputPort, InputPort, OutputPort}
+import wom.graph.GraphNodePort.{InputPort, OutputPort}
 
-class InstantiatedExpression private(val expression: WomExpression, val womReturnType: WdlType, val inputMapping: Map[String, InputPort]) {
+class InstantiatedExpression (val expression: WomExpression, val womReturnType: WdlType, val inputMapping: Map[String, InputPort]) {
```

Comment:
Hmm... I had this private to avoid people (by which I mean mostly myself) accidentally constructing invalid ones of these (e.g. by calling the constructor without validating the mapping). I *think* I'd prefer to have the verification here locally, rather than making this public

Reply:
Ok I had moved to bring `linkWithInputs` to `WomExpression`, but I'll put it back